pkg/blackduck/ctl_blackduck.go

```diff
@@ -139,6 +141,19 @@ func (ctl *Ctl) CheckSpecFlags(flagset *pflag.FlagSet) error {
 		cobra.MarkFlagRequired(flagset, "external-postgres-admin-password")
 		cobra.MarkFlagRequired(flagset, "external-postgres-user-password")
 	}
+
+	setStateToDbMigrate := flagset.Lookup("state-db-migrate").Changed
```

Comment:
With only a --migration (whatever) option on create we shouldn't need this logic, right?